cmd/oras/internal/display/metadata/tree/discover.go

```diff
@@ -74,6 +75,17 @@ func (h *discoverHandler) OnDiscovered(referrer, subject ocispec.Descriptor) err
 			referrerNode.AddPath(strings.TrimSpace(string(bytes)))
 		}
 	}
+
+	if len(referrer.Annotations) > 0 {
+		annotationsNode := referrerNode.Add("[annotations]")
+
+		for k, v := range referrer.Annotations {
+			coloredAnnotation := color.New(color.FgYellow).Sprintf("%s: %s", k, v)
+			annotationsNode.Add(coloredAnnotation)
+		}
+	} else {
+		referrerNode.Add(color.New(color.FgYellow).Sprint("[annotations] None"))
```

Comment:
We should use different colors for different types of node, such as artifact type, digest, etc. It would be similar to the screenshot in [this comment](https://github.com/oras-project/oras/issues/1534#issuecomment-2672059893).